### DISCUSSION TITLE
Where can I find a list of the color/size variables for use in my own classes?

### DISCUSSION THREAD

**Discussion by GlacierFox** [microsoft/fluentui-blazor#1738] [Q&A] [answered]
I'm looking for the Fluent color/size etc variables used by some of the other classes in the default app.css file.
I can see variables like:

```
font-size: var(--type-ramp-base-font-size);
line-height: var(--type-ramp-base-line-height);
```
and
```
color: var(--neutral-foreground-rest);
border-right: 1px solid var(--neutral-stroke-divider-rest);
```

Where can I find these so I can match sizing and colors to my own custom classes?

Thanks!

**Comment by Eagle3386** [comment]
@dvoituron This is pretty much the same question that I also asked in #1280 - would you mind answering it?
Because I _literally_ search through every file & found no such snippet!? 🤔😞

**Reply by dvoituron** [reply, marked as answer]
> @dvoituron This is pretty much the same question that I also asked in #1280 - would you mind answering it? Because I _literally_ search through every file & found no such snippet!? 🤔😞

These CSS variables are defined by the FluentUI Web Components (not by our code). The easy way to find these variables is to go to in the Browser DevTools.
![image](https://github.com/user-attachments/assets/711d8932-f685-4596-b38a-5ee2abc86f5f)

**Reply by Eagle3386** [reply]
Thanks so, so much, @dvoituron! ❤️
If possible, please mark your reply as answer, because I can't.
